damus/Models/Mentions.swift

```diff
@@ -10,13 +10,16 @@ import Foundation
 enum MentionType {
     case pubkey
     case event
+    case uri // uri mention type is used only when writing notes
     
-    var ref: String {
+    var ref: String? {
         switch self {
         case .pubkey:
             return "p"
         case .event:
             return "e"
+        case .uri:
+            return nil
```

Comment:
I don't think this is the right way to do it. we should be choosing pubkey or event depending on what's after the `nostr:`

Reply:
Fixed in 120d6bd049bda8ba6c363ef9391f88518375df3b